scripts/import-weekly-managed.sh

```
@@ -0,0 +1,243 @@
+# Copyright Contributors to the Open Cluster Management project
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)